src/smc-util/db-schema.coffee

```diff
@@ -821,6 +824,7 @@ schema.projects =
                 action_request : null   # last requested action -- {action:?, time:?, started:?, finished:?, err:?}
                 course         : null
                 compute_image  : 'latest'
+                academic       : {}
```

Comment:
Note that action_request is also an object above, as is course, but the default returned value is null, not {}.  I think that would be better for consistency.